spec/models/adventure_spec.rb

```diff
@@ -6,16 +6,20 @@
   it { should have_many(:bucket_list_adventures).dependent(:destroy) }
   it { should have_many(:users) }
 
-  it { should have_valid(:name).when('Visit Sue', 'Eat alligator') }
-  it { should have_valid(:address).when('Paris, France', '33 Harrison Ave, Boston') }
+  it { should have_valid(:name).when("Visit Sue", "Eat alligator") }
+  it { should have_valid(:address).when("Paris, France", "1 A St., NYC") }
 
   it { should have_valid(:latitude).when(40.7127837, 39.904211) }
-  it { should_not have_valid(:latitude).when('cat', '.') }
-  it { should validate_numericality_of(:latitude).is_greater_than_or_equal_to (-180) }
-  it { should validate_numericality_of(:latitude).is_less_than_or_equal_to (180) }
+  it { should_not have_valid(:latitude).when("cat", ".") }
+  it { should validate_numericality_of(:latitude)
+    .is_greater_than_or_equal_to(-180) }
```

Comment:
Expression at 15, 40 should be on its own line.<br>Place the . on the previous line, together with the method call receiver.